content/docs/command-reference/remote/modify.md

```diff
@@ -334,7 +334,7 @@ storage. Whether they're effective depends on each storage platform.
 - `account_name` (required) - storage account name
 
   ```dvc
-  $ dvc remote modify myremote account_name 'myuser'
+  $ dvc remote modify myremote account_name 'your_storage_account_name'
```

Comment:
Why do we change this? The param is already called `account_name`. Does `your_storage_account_name` as sample value add anything? Is there something wrong with `myuser` (consistent with many other existing samples).

Reply:
Hi @jorgeorpinel 
To be this was quite confusing at first as I thought `acount_name 'myuser'` referst to my azure account name and my username in azure. Whereas in fact `account_name` is the name of the storage account one is trying to access. 
But I agree that the snake_case is a bit too much here ;-)

Reply:
OK then "myuser" is incorrect it seems, as there's a separate username for Azure (not the same as account name)...

Maybe `myaccount`?